examples/playground/panel-lite-editor.js

```diff
@@ -0,0 +1,131 @@
+const appContainer = "app-container"
+const codeEditor = "code-editor"
+
+const welcomeExample=`\
+import panel as pn
+pn.extension()
+
+pn.pane.Markdown("Welcome! Try clicking the \`run\` button.").servable()
+`
+
+const buttonExample=`\
+import panel as pn
+pn.extension(design="material")
+button = pn.widgets.Button(name="Click Me")
+component = pn.Row(button, pn.bind(lambda c: c, button.param.clicks))
+
+component.servable()
+`
+
+const matplotlibExample=`
+import panel as pn
+
+import numpy as np
+
+from matplotlib.figure import Figure
+from matplotlib import cm
+from matplotlib.backends.backend_agg import FigureCanvas  # not needed for mpl >= 3.1
+
+Y, X = np.mgrid[-3:3:100j, -3:3:100j]
+U = -1 - X**2 + Y
+V = 1 + X - Y**2
+speed = np.sqrt(U*U + V*V)
+
+fig0 = Figure(figsize=(8, 6))
+ax0 = fig0.subplots()
+FigureCanvas(fig0)  # not needed for mpl >= 3.1
+
+strm = ax0.streamplot(X, Y, U, V, color=U, linewidth=2, cmap=cm.autumn)
+fig0.colorbar(strm.lines)
+
+pn.pane.Matplotlib(fig0, dpi=144).servable()
+`
+
+const perspectiveExample=`
+import panel as pn
+import pandas as pd
+
+import random
+from datetime import datetime, timedelta
+
+pn.extension("perspective")
+
+data = {
+    "int": [random.randint(-10, 10) for _ in range(9)],
+    "float": [random.uniform(-10, 10) for _ in range(9)],
+    "date": [(datetime.now() + timedelta(days=i)).date() for i in range(9)],
+    "datetime": [(datetime.now() + timedelta(hours=i)) for i in range(9)],
+    "category": [
+        "Category A",
+        "Category B",
+        "Category C",
+        "Category A",
+        "Category B",
+        "Category C",
+        "Category A",
+        "Category B",
+        "Category C",
+    ],
+}
+df = pd.DataFrame(data)
+
+pn.pane.Perspective(df, width=1000).servable()
+`
+
+const fastTemplateExample=`\
```

Comment:
Get the `fastTemplateExample` working

- [ ] Most times nothing happens when I click the button. 
- [ ] I can only run this example once. See https://github.com/holoviz/panel/issues/5810
- [ ] I have seen this example render in different ways. For example sometimes there is an extra button in the header.